app/layout.tsx

```diff
@@ -4,30 +4,52 @@ import { Footer } from '@/components/footer';
 import { CartProvider } from '@/components/cart/cart-provider';
 
 import './globals.css';
+import { apiRequest } from '@/utils/api-request';
+import { FetchLandingPageDocument, FrontPageMetadataDocument } from '@/generated/graphql';
 
 const manrope = Manrope({ subsets: ['latin'] });
 
-export const metadata: Metadata = {
-    title: 'Furnitut: Furniture for the future',
-    description: 'Furnitut is a boilerplate created by Crystallize using Next.js.',
-};
+export async function generateMetadata(): Promise<Metadata> {
+    const { data } = await apiRequest(FrontPageMetadataDocument);
+
+    const meta = data.browse?.landingPage?.hits?.[0]?.meta;
+    const title = meta?.title;
+    const description = meta?.description[0].textContent;
+    const image = meta?.image?.[0];
+
+    return {
+        title: `${title} | Furnitut`,
+        description,
+        creator: 'Crystallize Team',
+        openGraph: {
+            title: `${title} | Furnitut`,
+            description,
+            images: [{
+                url: image?.url ?? '',
+                alt: image?.altText ?? '',
+                height: image?.height ?? 0,
+                width: image?.width ?? 0,
+            }],
+        },
+    };
+}
 
 type LayoutProps = { children: React.ReactNode };
 
 export default async function Layout({ children }: LayoutProps) {
     return (
         <html lang="en">
-            <head>
-                <meta charSet="utf-8" />
-                <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-                <link rel="icon" href="/favicon.ico" />
-            </head>
-            <body className={`${manrope.className} bg-soft`}>
-                <CartProvider>
-                    {children}
-                    <Footer />
-                </CartProvider>
-            </body>
+        <head>
+            <meta charSet="utf-8" />
+            <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+            <link rel="icon" href="/favicon.ico" />
+        </head>
+        <body className={`${manrope.className} bg-soft`}>
+        <CartProvider>
+            {children}
+            <Footer />
+        </CartProvider>
+        </body>
```

Comment:
Hmm, this has to be nested ... there is something wrong with your prettier set up